applications/adv_networking_bench/adv_networking_bench_default_tx_rx_multi_q_hds.yaml

```diff
@@ -0,0 +1,334 @@
+scheduler:
+  check_recession_period_ms: 0
+  worker_thread_number: 5
+  stop_on_deadlock: true
+  stop_on_deadlock_timeout: 500
+  max_duration_ms: 10000
+advanced_network:
+  cfg:
+    version: 1
+    manager: dpdk
+    master_core: 3
+    debug: false
+    log_level: info
+    tx_meta_buffers: 4096
+    rx_meta_buffers: 4096
+    memory_regions:
+    - name: Headers_RX_CPU
+      kind: huge
+      affinity: 0
+      access:
+      - local
+      num_bufs: 51200
+      buf_size: 42
+    - name: Context_RX_CPU
+      kind: huge
+      affinity: 0
+      access:
+      - local
+      num_bufs: 10240
```

Comment:
Low importance, but isn't that number of buffers unnecessarily high if the batch_size is `1` ? Our rule of thumb in the docs is 5x batch size.

Reply:
k

Reply:
https://jirasw.nvidia.com/browse/CLARAHOLOS-1907